oada/services/permissions-handler/package.json

```diff
@@ -4,22 +4,22 @@
   "description": "",
   "main": "server.js",
   "scripts": {
+    "start": "ts-node --files server.ts",
     "test": "DEBUG=* mocha test/permissions-handler.js"
   },
   "author": "",
   "license": "ISC",
+  "devDependencies": {
+    "@types/mocha": "^8.2.0",
+    "type-is": "^1.6.15"
+  },
   "dependencies": {
-    "axios": "^0.21.0",
-    "bcryptjs": "^2.4.3",
-    "bluebird": "^3.5.0",
-    "debug": "^4.3.1",
-    "lodash": "^4.17.4",
-    "moment": "^2.18.1",
-    "nconf": "^0.11.0",
     "@oada/lib-arangodb": "^0.0.2",
     "@oada/lib-config": "^0.0.1",
-    "@oada/lib-kafka": "^1.0.0",
-    "randomstring": "^1.1.5",
-    "type-is": "^1.6.15"
+    "@types/type-is": "^1.6.3",
```

Comment:
I think you have these backwards. The `@types` should be the devDependency.

Reply:
Your right, I actually don't know what `type-is` is, so must have somehow accidentally done this.